database/processes/screens/default-email-task-notification.json

```diff
@@ -111,7 +111,7 @@
                         "config":{
                             "icon":"fas fa-pencil-ruler",
                             "label":null,
-                            "content":"<p style=\"font-size: 36px;\"><strong>You just been assigned a task: {{element_name}}<\/strong><\/p>",
+                            "content":"<p style=\"color: #20242a; text-align: center; font-family: Inter, sans-serif; font-size: 24px; font-style: normal; font-weight: 600; line-height: 32px; letter-spacing: -0.24px;\"><strong>You have been assigned a task: {{_task_name}}<\/strong><\/p>",
```

Comment:
the font-size in figma is 36px

Reply:
@fagubla
The figma show this information:
<img width="1196" height="702" alt="image" src="https://github.com/user-attachments/assets/fae46d32-dc6e-4995-9302-4236c025a348" />
Regards,